stdlib/asm/math/u64.masm

```diff
@@ -427,9 +427,9 @@ end
 #! Stack transition looks as follows:
 #! [b_hi, b_lo, a_hi, a_lo, ...] -> [c_hi, c_lo, ...], where c = a // b
 export.unchecked_div
-    adv.u64div          # inject the quotient and the remainder into the advice tape
+    adv.u64div          # inject the quotient and the remainder into the advice stack
 
-    adv_push.2          # read the quotient from the advice tape and make sure it consists of
+    adv_push.2          # read the quotient from the advice stack and make sure it consists of
```

Comment:
Let's change the comments to:

* "push the quotient and the remainder onto the advice stack"
* "pop the quotient from the advice stack and make sure it consists of"

Separately, I wonder if we should rename `adv_push` -> `adv_pop` (since the operation pops a value from the advice stack). Let's create an issue for this?

Reply:
`adv_loadw`, renamed to `adv_popw`, seems to have a counter-intuitive behavior.

The other instructions will pop values from the advice provider onto the operand stack, while this one will "override" the top 4 elements. Example:

Given we have an advice stack described as  `[a, b, c, d, ...]`, and operand stack as `[m, n, o, p, ...]`

`adv_popw` will result int `[d, c, b, a, ...]`. However, I think it would be more intuitive to `[d, c, b, a, m, n, o, p, ...]`. It's naturally not related to this PR, but I could open a new one to review this implementation